src/main/java/app/entities/unitconverter/TimeConverter.java

```diff
@@ -0,0 +1,106 @@
+package app.entities.unitconverter;
+
+public class TimeConverter
+{
+    public double secondsToMinutes(double seconds)
+        {
```

Comment:
this indentation is something that is usually addressed in these PRs.

We did however not settle on a code convention, but when we do this will not be it 😉